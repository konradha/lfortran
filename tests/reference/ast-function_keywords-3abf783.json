{
    "basename": "ast-function_keywords-3abf783",
    "cmd": "lfortran --indent --fixed-form --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form/function_keywords.f",
    "infile_hash": "93e87638f8bc162a984d073c23fa0b6e7df7f2feb462866a95993366",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "ast-function_keywords-3abf783.stderr",
    "stderr_hash": "dc69f90826b8655526c9919a26825d60d4fb79b4aa744951a1938e88",
    "returncode": 1
}