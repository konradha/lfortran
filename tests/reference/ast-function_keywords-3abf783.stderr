runtime_error: malloc failed.
